DEPLOYING.md

```diff
@@ -47,6 +47,56 @@ To change the default env vars used throughout the [CKAN configuration file](./d
 vim docker-compose/ckan-secrets.sh
 ```
 
+Also, set or adjust deployment related environment variables in the [docker-compose.yaml](./docker-compose.yaml) and [.docker-compose.vital-strategies-theme.yaml](./.docker-compose.vital-strategies-theme.yaml) Few of them worth to talk about:
+
+Database passwords - This password will be set to the databases so make sure to update and choose them carefully
+```
+# in docker-compose.yaml and docker-compose-db.yaml
+POSTGRES_PASSWORD=123456
+DATASTORE_RO_PASSWORD=123456
+DATASTORE_PUBLIC_RO_PASSWORD=123456
+```
+
+Database URL - this URLs (URIs) will be used by CKAN to connect databases. Update them according to passwords set above
+```
+# in docker-compose/ckan-secrets.sh
+SQLALCHEMY_URL=postgresql://ckan:123456@db/ckan
+CKAN_DATASTORE_WRITE_URL=postgresql://postgres:123456@datastore-db/datastore
+CKAN_DATASTORE_READ_URL=postgresql://readonly:123456@datastore-db/datastore
+```
+
+You can increase number of [Gunicorn](https://gunicorn.org/) workers that runs CKAN
+```
+# in docker-compose.yaml
+GUNICORN_WORKERS=2
+```
+
+Modify storage path where CKAN resources will be store
+```
+# in docker-compose.yaml
+CKAN_STORAGE_PATH=/var/lib/ckan/data
+```
+
+Optional: You can also use S3 filestore instead for saving resorces. For that you will need to install and enable [S3Filestore extension](https://github.com/datopian/ckanext-s3filestore) plugin. You will need following env variables
+```
+# docker-compose/ckan-secrets.sh
+AWS_ACCESS_KEY_ID=YourSecretKey
+AWS_SECRET_ACCESS_KEY=YourSecretAccessKey
+```
+
+SMTP service credentials - this credentials are used by CKAN to send email from Eg: password reset
```

Comment:
We might need to mention at the beginning of the doc that you need a SMTP server for the deployment.